src/Orbits.jl

```diff
@@ -1,194 +1,385 @@
+function RV_semiamplitude_K1(;m_1, m_2, P, e, i)
+    return (m_2*sin(i))*cbrt(2*π*cgrav/(P*day_in_sec)*m_sun/(m_2+m_1)^2)/sqrt(1-e^2)/(km) #semi amplitude of RV variation in km/s
+end
+
+
+###########################################################
+### 
+### Post kick orbital properties, for circular pre-explosion orbits
+### 
+###########################################################
+
+"""
+    post_supernova_circular_orbit_a(;m_1i, m_2i, a_i, m_1f=-1, m_2f, vkick=0, θ=0, ϕ=0, vimp=0)
+
+Compute post-kick properties for a circular pre-explosion orbit using equations 
+from Tauris & Takens (1999), except that here we have M2 as the exploding star.
+
+# Arguments:
+- m_1i:  pre-explosion  mass of non-exploding component [Msun]           
+- m_2i:  pre-explosion  mass of exploding component     [Msun]       
+- a_i:   pre-explosion orbital separation               [cm]
+- m_1f:  post-explosion mass of non-exploding component [Msun]           
+- m_2f:  post-explosion mass of exploding component     [Msun]   
+- vkick: kick velocity                                  [km/s] 
+- θ:     polar kick angle (away from e_par)             [rad]
+- ϕ:     azimuthal kick angle (off of e_perp)           [rad]
+- vimp:  imparted kick velocity on companion            [km/s]     
     
-#Output:
-- afinal: post-explosion orbital separation          [cm]
-- efinal: post-explosion excentricity                [dimensionless]
+# Output:
+- a_f: post-explosion orbital separation                [cm]
+- e_f: post-explosion excentricity                      [dimensionless]
 """
-function post_kick_parameters_a(a,mtilde,vkick_div_vrel,theta,phi)
-    xi = (1+vkick_div_vrel^2+2*vkick_div_vrel*cos(theta))/(mtilde)
-    Q = xi - 1 - (vkick_div_vrel*sin(theta)*cos(phi))^2/(mtilde)
+function post_supernova_circular_orbit_a(;m_1i, m_2i, a_i, m_1f=-1, m_2f, vkick=0, θ=0, ϕ=0, vimp=0)
+    if m_1f == -1
+        m_1f = m_1i
+    end
+    mtilde = (m_1f+m_2f)/(m_1i+m_2i) 
+    v_rel = average_orbital_velocity(;m_1=m_1i, m_2=m_2i, a=a_i)
+    α = vkick/v_rel
+    β = vimp/v_rel
+    #vkick_div_vrel = vkick/v_rel
+    #vimp_div_vrel = vimp/v_rel
+    # convert trig functions to vars
+    cosθ = cos(θ)
+    sinθ = sin(θ)
+    cosϕ = cos(ϕ)
+    sinϕ = sin(ϕ)
+
+    #println("==> ", cosθ, " ", cosϕ)
+    ξ = (1 + α^2 + β^2 + 2*α*cosθ - 2*β*sinθ*cosϕ)/(mtilde)
+    #println(ξ)
+    #ξ = f_ν*g_ν^2*M_i/M_f*(1+α^2+β^2+2*(-h_ν*β*(1+α*cosθ)+α*cosθ-j_ν*β*α*sinθ*cosϕ))
+
 
-    if (xi>2)
+
+    if (ξ>2)
         return (NaN, NaN)
     end
-    afinal = a/(2-xi)
-    efinal = sqrt(1+(xi-2)*(Q+1))
-    return (afinal,efinal)
+    tanγ = (vkick*sinθ*cosϕ - vimp)/sqrt((v_rel + vkick*cosθ)^2 + vkick^2*sinθ^2*sinϕ^2)
+    cosγ = 1/sqrt(1+tanγ^2)
+
+    # Orbital parameters
+    a_f = a_i/(2-ξ)
+    e_f = sqrt(1 + ξ*(ξ-2)*cosγ^2)
+    #println("af=",a_f)
+
+    #Q = (ξ-1) - (vkick_div_vrel*sinθ*cosϕ - vimp_div_vrel)^2/(mtilde) 
+    #e_f_old = sqrt(1+(ξ-2)*(Q+1)) # RTW where does this come from?
```

Comment:
This is equation (6) of Tauris+ 1999
https://ui.adsabs.harvard.edu/abs/1999MNRAS.310.1165T/abstract
You can implement it like they have it there and compare to the general kick function. It would be nice if we can write things in a way that is equivalent to Tauris+ 1999, such that the general kick reduces to that circular solution for e=0. This is the case for the orbital separation right now.

Reply:
I disagree, I think we should be writing it so that it matches with what we have in the overleaf, since that's what we are directly comparing to. We can test that the circular case matches the general for e=0, but beyond that I don't think it's important that the variables match exactly.

Reply:
I mean here that it would be nice if our own writeup trivially reduces to Tauris, Fender+ 1999.